src/main/java/de/tum/in/www1/artemis/service/notifications/GroupNotificationService.java

```diff
@@ -256,8 +256,8 @@ public void notifyTutorAndEditorAndInstructorGroupAboutNewReplyForExercise(Post
         notifyGroupsWithNotificationType(new GroupNotificationType[] { TA, EDITOR, INSTRUCTOR }, NEW_REPLY_FOR_EXERCISE_POST, post, course, answerPost.getAuthor());
     }
 
-    public void notifyTutorAndEditorAndInstructorGroupAboutNewFeedbackRequest(Exercise exercise) {
-        notifyGroupsWithNotificationType(new GroupNotificationType[] { TA, EDITOR, INSTRUCTOR }, NEW_MANUAL_FEEDBACK_REQUEST, exercise, null, null);
+    public void notifyTutorAndEditorGroupAboutNewFeedbackRequest(Exercise exercise) {
+        notifyGroupsWithNotificationType(new GroupNotificationType[] { TA, EDITOR }, NEW_MANUAL_FEEDBACK_REQUEST, exercise, null, null);
```

Comment:
The job of editors is the creation of new exercises. Should they be notified here?
In case these people create exercises and should also assess submissions, you can assign them both roles in the course

Reply:
makes sense